docs/src/pages/components/geo/fragments/marker-moving.react.mdx

```diff
@@ -0,0 +1,30 @@
+```jsx
+import { AmplifyMap } from '@aws-amplify/ui-react';
+import { Amplify } from 'aws-amplify';
+import { Marker } from 'react-map-gl';
+
+import '@aws-amplify/ui-react/styles.css';
+
+import awsExports from './aws-exports';
+
+Amplify.configure(awsExports);
+
+export default function MapWithMovingMarker() {
+  const [{ latitude, longitude }, setMarkerLocation] = useState({
+    latitude: 40,
+    longitude: -100,
+  });
+
+  const updateMarker = () =>
+    setMarkerLocation({ latitude: latitude + 5, longitude: longitude + 5 });
+
+  return (
+    <>
+      <button onClick={updateMarker}>Move Marker</button>
```

Comment:
Perhaps worth demonstrating how to use the `Button` component from `@aws-amplify/ui-react`?

Reply:
Sure, I'll add it in.